docs/sources/datasources/prometheus.md

```diff
@@ -141,6 +141,11 @@ There are two syntaxes:
 Why two ways? The first syntax is easier to read and write but does not allow you to use a variable in the middle of a word. When the _Multi-value_ or _Include all value_
 options are enabled, Grafana converts the labels from plain text to a regex compatible string. Which means you have to use `=~` instead of `=`.
 
+### Ad hoc filters variable
+
+Prometheus supports the special `Ad hoc filters` variable type. This variable allows you to specify any number of label/value filters on the fly. These filters will automatically
+be applied to all your Prometheus queries.
```

Comment:
```suggestion
applied to all your Prometheus queries.
```

Reply:
This is probably a better path:https://grafana.com/docs/grafana/latest/variables/variable-types/add-ad-hoc-filters/